cypress-custom/integration/fee.test.ts

```diff
@@ -21,42 +22,47 @@ describe('Fetch and persist fee', () => {
   beforeEach(() => {
     cy.visit('/swap')
 
-    // Alias localStorage
     cy.window()
       .then(window => window.localStorage)
       .as('localStorage')
   })
 
   it('Persisted when selecting a token', () => {
-    const TOKEN = 'ETH'
-    const NETWORK = ChainId.RINKEBY.toString()
-    // GIVEN: A fee for a token is not in the local storage
-    // WHEN: When the user select this token
+    const DAI = '0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735'
```

Comment:
what you recommended in an earlier PR can be done here via `context`

e.g

```ts
context('Fee: localStorage', () => {
  const DAI = '...'
  const foo = '...'
  it( /* All tests as normal can reference context constants */ , () => { ... })
})

Reply:
mm, I didn't understand. You are suggesting to use the context for defining the token? 
We could do that, but it feels like over engineering a const we can use directly imo. Maybe I'm missing sth

Reply:
Also context is not so typescript friendly. You rely wither in `this.stuff` or in `cy.get('@stuff')`

Reply:
all good the approach in the next waterfall above this makes this comment irrelevant. just figured I would show you that `context` is available for giving tests contexts and creates oragnised nesting views in the GUI mode of cypress